config/pipeline.yaml

```diff
@@ -23,6 +23,9 @@ api_configs:
   early-access:
     url: https://kernelci-api.eastus.cloudapp.azure.com
 
+  k8s-host:
+    url: http://kernelci-api:8001
```

Comment:
How is this going to resolve when using the command locally on the actual host?  Is this meant to only be accessible from within the cluster?

Reply:
Yes, `k8s-host` is for accessing API within the cluster.
To use it locally, we need to `post-forword` API service and then `http://localhost:8001` would work fine.
The command for port forward has been mentioned [here](https://github.com/kernelci/kernelci-api/blob/main/kube/minikube/README.md).

Reply:
Right, and then that would require a different URL in the YAML config.

Another way to do it I guess would be to define a service, for example:
```yaml
apiVersion: apps/v1
kind: Deployment
metadata:
  name: web
spec:
  replicas: 1
  selector:
    matchLabels:
      app: web
  template:
    metadata:
      labels:
        app: web
    spec:
      containers:
      - name: web
        image: nginx:1.21.3
        ports:
        - containerPort: 80
---
apiVersion: v1
kind: Service
metadata:
  name: web
spec:
  type: LoadBalancer
  ports:
  - port: 80
  selector:
    app: web
```